webapp/src/components/ErrorAlert.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <v-alert :value="!!(errors && errors.length)" type="error">
```

Comment:
add style `overflow: auto` which is otherwise lost in one of the deletions furthers down.

Original commit: https://github.com/desec-io/desec-stack/commit/a5d671269a645851650aa949362d1e8254052172#diff-5d31baaa7fb6173d2864fd65d9ef249d4c626c7975146684559c155f30e5809aR104

Reply:
done